### DISCUSSION TITLE
does but get this

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#299] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
It does work because my irl friend did and you need a file for it and that's why it's not working and you can even ask him if you need a file for this

**Comment by PickleIsDaBest** [comment]
What do you mean?
Can you explain more?

**Reply by ghost** [reply]
k so the file is the unblocker because of the code it carry's and that how it unblockeds every website but the downfall is that i'm still looking for the file so i give it to every maybe but i'm not the one who made the thing and idk what code it carry's might be C++ or java or python or something else that i don't know but i'm sorry responded you to you so late hope this is making sense to you i will give you the file when i find it and to everyone have a nice day :) :) :) :) :)

**Comment by bartholemuew** [comment]
how much u wanna bet he's on a lower version 💀 bro 100% lagging fr

**Reply by ghost** [reply]
me?

**Comment by Aka-but-nice** [comment]
Bro, I speak English what are you trying to say? I can't understand.

**Reply by ghost** [reply]
sorry if i don't make sense i was typing fast and when i typed to fast the words does not make any sense :P
